templates/single.html

```diff
@@ -2,9 +2,9 @@
 
 <!-- wp:group {"tagName":"main"} -->
 <main class="wp-block-group"><!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group"><!-- wp:post-title {"level":1,"align":"wide","style":{"spacing":{"margin":{"bottom":"var(--wp--preset--spacing--70)"}}}} /-->
+<div class="wp-block-group"><!-- wp:post-title {"level":1,"align":"wide"} /-->
 
-<!-- wp:post-featured-image {"align":"wide","style":{"spacing":{"margin":{"bottom":"var(--wp--preset--spacing--70)"}}}} /-->
```

Comment:
Why remove this one but it's fine on others like `footer-default.html`?

Reply:
I feel like this should be handled by block gap with the flow of the document, whereas the header and footer are outside that flow.